### DISCUSSION TITLE
SPEEDYBEE F7 BZ- pin destroyed

### DISCUSSION THREAD

**Discussion by alex-kwinto** [iNavFlight/inav#6436] [Q&A] [answered]
Hi,

I accidentally destroyed pin BZ- on my SPEEDYBEE F7. Is someone know where i can solder to? 
Also i was considering custom firmware like to switch pins PC13 and PC14 in target.h file.

Any thoughts?

Thanks in advance!

**Comment by DzikuVx** [comment, marked as answer]
`BZ-` leads to a small MOSFET driver. It's not directly connected to any of the pins on the MCU. Find the buzzer driver and solder to drain pin on it

**Comment by alex-kwinto** [comment]
Just for integrity. Here is the proper solder place

![image](https://user-images.githubusercontent.com/63970492/102799085-aff55100-43ba-11eb-96ad-fe007f8b4138.png)

Issue can be closed
